frontend/css/layout.css

```diff
@@ -14,21 +16,18 @@ a {
 
 .account {
     justify-content: flex-end;
-    padding-right: 20px;
     border-bottom: 0.5px solid lightgray;
```

Comment:
Again I know this is still in progress 🙂 Just adding a note about moving the spacing and colour into variables later on.

Reply:
Agreed to fix in later PR.